tox.ini

```diff
@@ -0,0 +1,58 @@
+[tox]
+envlist =
+    flake8
+    isort
+    py{35,36}-dj21-cms36
+
+skip_missing_interpreters=True
+
+[flake8]
+max-line-length = 119
+exclude =
+    *.egg-info,
+    .eggs,
+    .git,
+    .settings,
+    .tox,
+    build,
+    data,
+    dist,
+    docs,
+    *migrations*,
+    requirements,
+    tmp
+
+[isort]
+line_length = 79
+skip = manage.py, *migrations*, .tox, .eggs, data
+include_trailing_comma = true
+multi_line_output = 5
+not_skip = __init__.py
+lines_after_imports = 2
+default_section = THIRDPARTY
+sections = FUTURE, STDLIB, DJANGO, CMS, THIRDPARTY, FIRSTPARTY, LIB, LOCALFOLDER
+known_first_party = aldryn_django
+known_cms = cms, menus
+known_django = django
+
+[testenv]
+deps =
+    -r{toxinidir}/tests/requirements.txt
+    dj21: Django>=2.1,<2.2
+    cms34: django-cms>=3.4,<3.5
```

Comment:
do we need this here? You are using only cms36 in tox file
cms 34 and 35 doesn't support Django 2.1 so we can remove them from this file